pkg/rslog/logger_impl.go

```diff
@@ -32,14 +32,16 @@ func (f *LoggerFactoryImpl) DefaultLogger() Logger {
 	return lgr
 }
 
-type LoggerImpl struct {
-	*logrus.Logger
+func (f *LoggerFactoryImpl) TerminalLogger(l LogLevel) Logger {
+	lgr, _ := NewLoggerImpl(LoggerOptionsImpl{
```

Comment:
just for my own understanding - why do we not care about the possible `err` here?

Reply:
Same question, I assume that because this time it is using a StderrOutputBuilder, it isn't possible to error... but want to be sure. Also, if that's the case, having a comment like `// Ignoring error because ...` will be good for future reference :)

Reply:
Not the author but I think it is like that for two reasons:
1. It is a counterpart for `DefaultLogger`, and thus conforms with that, which also doesn't return any errors
2. Taking a look at the implementation from `NewLoggerImpl`, the error returned is from a call to `(*StderrOutputBuilder).Build`, which never returns any errors, so it's all good

Reply:
Exactly, I'm ignoring the error here because only the builder could return an error, and the `StderrOutputBuilder` doesn't return errors. But thinking better, It would be good to check the errors because we would be already covered in case of any change to the `NewLoggerImpl`. WDYT?

Reply:
@eduardocfalcao I think if we do that for `TerminalLogger`, maybe we would want to do that for `DefaultLogger` as well. Even because `DefaultLogger` seems able to return an error, and I think we're currently ignoring it. I understand that would be a lot less convenient so I would agree of only doing so for `TerminalLogger`.

Reply:
Yeah, we only worry about the errors when creating outside the `rslog` package. By now, I think I will only add comments to explain why we are ignoring the errors here.

It can be changed, but I'm moving the discussion for later.